src/cli/install.rs

```diff
@@ -198,27 +198,20 @@ fn _run(cmd: &Command) -> anyhow::Result<()> {
             .with_context(|| format!("failed to write env file {:?}", settings.env_file))?;
     }
 
+    // We no longer support migration from the old layout.
     let base = home_dir()?.join(".edgedb");
     let new_layout = if base.exists() {
         eprintln!(
             "\
                 {BRANDING_CLI} no longer uses '{}' to store data \
                 and now uses standard locations of your OS. \
+                If you have migrated, you can safely ignore this message \
+                and remove the directory. If you have not migrated, you can \
+                download and run an older version of `edgedb-cli` to migrate. \
```

Comment:
I think it's pretty difficult for someone to "run an older version of `edgedb-cli`" without asking us directly how to do this. Not really sure what to say about this other than maybe we keep the old code around forever or consider this a breaking change and don't point someone at the impossible task of figuring out how to download an older version.

Reply:
I could potentially just remove this check as I think it's pretty unlikely anyone will end up in this situation.

Reply:
Yeah, I'm down with that.